docs/reference/ccr/index.asciidoc

```diff
@@ -266,12 +266,17 @@ retaining these soft deletes up to configurable limits, the history of
 operations can be retained on the leader shards and made available to the
 follower shard tasks as it replays the history of operations.
 
-The <<ccr-index-soft-deletes-retention-period,`index.soft_deletes.retention_lease.period`>> setting defines the
-maximum time to retain a shard history retention lease before it is
-considered expired. This setting determines how long the cluster containing
-your leader index can be offline, which is 12 hours by default. If a shard copy
-recovers after its retention lease expires, then {es} will fall back to copying
-the entire index, because it can no longer replay the missing history.
+The <<ccr-index-soft-deletes-retention-period,`index.soft_deletes.retention_lease.period`>>
+setting defines the maximum time to retain a shard history retention lease
+before it is considered expired. This setting determines how long the cluster
+containing your follower index can be offline, which is 12 hours by default. If
```

Comment:
NB `s/leader/follower/`, if the leader is offline then there are no missing operations. I guess it's more subtle than that too, not sure we really need to clarify it completely tho.

Reply:
This text is clear and understandable. No need to change that, in order to list all possible reasons why ops might still exist in leader cluster.